docs/IDE-Setup.md

```diff
@@ -176,6 +176,18 @@ Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Se
 
 Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Setup.md#bazel-cannot-read-environment-variable).
 
+#### Robolectric Errors in Android JUnit Tests
+
+The likely culprit is an incompatability between the [version of Robolectric](https://github.com/PolymerLabs/arcs/blob/master/WORKSPACE#L48) and the version of Java on your machine. 
+
+We recommend that you use Java 8 or 11. Other Java versions have be associated with errors at the time of writing.
```

Comment:
nit: be -> been

also consider stating the current month/year where we say "time of writing" for clarity in xx months etc.